### ISSUE TITLE
Download action destroys json

### ISSUE THREAD

**Issue by PaulWoitaschek** [fastlane/fastlane#13392]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
The fastlane's download actions destroys json.

Here is a lane that demonstrates the problem:
```ruby
  lane :download_bug do
    result = download(url: 'https://gist.githubusercontent.com/PaulWoitaschek/24621899f04fcda04e4158ace5c6675a/raw/0587ea3e75b517cd0ad0af5524d4cb12e4592ef0/gistfile1.txt')
    puts result
  end
```

The url contains the following json: 
```json
{"a":1, "b":2}
```

This now puts 
```{"a"=>1, "b"=>2}``` which is no longer valid json.

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>[INSERT OUTPUT HERE]</pre>
</details>

### Environment

<details>
  <pre>[### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.104.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                                     | Version | Update-Status |
| ------------------------------------------ | ------- | ------------- |
| fastlane-plugin-run_tests_firebase_testlab | 0.2.0   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                        | Version      |
| ------------------------------------------ | ------------ |
| slack-notifier                             | 2.3.2        |
| atomos                                     | 0.1.3        |
| CFPropertyList                             | 3.0.0        |
| claide                                     | 1.0.2        |
| colored2                                   | 3.1.2        |
| nanaimo                                    | 0.2.6        |
| xcodeproj                                  | 1.6.0        |
| rouge                                      | 2.0.7        |
| xcpretty                                   | 0.3.0        |
| terminal-notifier                          | 1.8.0        |
| unicode-display_width                      | 1.4.0        |
| terminal-table                             | 1.8.0        |
| plist                                      | 3.4.0        |
| public_suffix                              | 2.0.5        |
| addressable                                | 2.5.2        |
| multipart-post                             | 2.0.0        |
| word_wrap                                  | 1.0.0        |
| tty-screen                                 | 0.6.5        |
| tty-cursor                                 | 0.6.0        |
| tty-spinner                                | 0.8.0        |
| babosa                                     | 1.0.2        |
| colored                                    | 1.2          |
| highline                                   | 1.7.10       |
| commander-fastlane                         | 4.4.6        |
| excon                                      | 0.62.0       |
| faraday                                    | 0.15.2       |
| unf_ext                                    | 0.0.7.5      |
| unf                                        | 0.1.4        |
| domain_name                                | 0.5.20180417 |
| http-cookie                                | 1.0.3        |
| faraday-cookie_jar                         | 0.0.6        |
| fastimage                                  | 2.1.3        |
| gh_inspector                               | 1.1.3        |
| json                                       | 2.1.0        |
| mini_magick                                | 4.5.1        |
| multi_json                                 | 1.13.1       |
| multi_xml                                  | 0.6.0        |
| rubyzip                                    | 1.2.2        |
| security                                   | 0.1.3        |
| xcpretty-travis-formatter                  | 1.0.0        |
| dotenv                                     | 2.5.0        |
| bundler                                    | 1.16.4       |
| faraday_middleware                         | 0.12.2       |
| naturally                                  | 2.2.0        |
| simctl                                     | 1.6.5        |
| uber                                       | 0.1.0        |
| declarative                                | 0.0.10       |
| declarative-option                         | 0.1.0        |
| representable                              | 3.0.4        |
| retriable                                  | 3.1.2        |
| mime-types-data                            | 3.2018.0812  |
| mime-types                                 | 3.2.2        |
| jwt                                        | 2.1.0        |
| memoist                                    | 0.16.0       |
| os                                         | 1.0.0        |
| signet                                     | 0.9.0        |
| googleauth                                 | 0.6.6        |
| httpclient                                 | 2.8.3        |
| google-api-client                          | 0.23.5       |
| emoji_regex                                | 0.1.1        |
| openssl                                    | 2.1.0        |
| date                                       | 1.0.0        |
| fileutils                                  | 1.0.2        |
| ipaddr                                     | 1.2.0        |
| fastlane-plugin-run_tests_firebase_testlab | 0.2.0        |
</details>

*generated on:* **2018-09-19**
</details>
[21:57:41]: Take notice that this output may contain sensitive information, or simply information that you don't want to make public.
[21:57:41]: Open https://github.com/fastlane/fastlane/issues/new to submit a new issue ✅
[ph1b@archlinux Yazio]$ 
</pre>
</details>

**Comment by heartofagoof** [comment]
When you download something it's parsed into a hash. I'd suggest doing 

```ruby
  lane :download_bug do
    result = download(url: 'https://gist.githubusercontent.com/PaulWoitaschek/24621899f04fcda04e4158ace5c6675a/raw/0587ea3e75b517cd0ad0af5524d4cb12e4592ef0/gistfile1.txt')
    puts result.to_json
  end
```

**Comment by KrauseFx** [comment]
Yep, thanks for the reply @heartofagoof, that is the valid approach, it's Ruby code 👍
